### ISSUE TITLE
tokio-runtime-worker

### ISSUE THREAD

**Issue by fatmacats** [biomejs/biome#6749]
### Environment information

```block
CLI:
  Version:                      1.9.4
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v22.2.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Error:                        Found an unknown key `noUndeclaredDependencies`.
  Status:                       Loaded with errors
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates\biome_service\src\settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.
StdIO connection closed
Exit code: 0
Command line: C:/GIT/s2968/fe01/node_modules/@biomejs/cli-win32-x64/biome.exe lsp-proxy --config-path C:\GIT\s2968\fe01

### Expected result

to start biome server

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
